Cargo.toml

```diff
@@ -20,34 +20,33 @@ maintenance = { status = "actively-developed" }
 
 [features]
 default = []
-google = ["base64", "google-pubsub1", "yup-oauth2"]
-mock = []
```

Comment:
why remove this feature?

Reply:
The `MockPublisher` implementation does not pull in any dependencies and is otherwise a very lean module. Ultimately if it ends up not being used, the compiler will nuke it out from final binary either way.

Furthermore, due to the fact that enabling the feature for testing will also enable it for regular builds, most of the users of this library will end up enabling `mock` regardless of whether they are compiling tests or something else.

All that considered I find this feature to add very little value and the cognitive overhead was significant.